recipes/r-tidyjson/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = '0.2.2.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tidyjson
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  url: https://github.com/sbilge/tidyjson/archive/v0.2.2.3.tar.gz
```

Comment:
This is not great. This is your personal fork of the repository.

What is the status of this package? Is colearendt the new maintainer? While a CRAN release is the best case scenario, at minimum it would be ideal if the maintainer tagged a release on the offical GitHub repository. I see you started the discussion in https://github.com/colearendt/tidyjson/issues/111

Have you confirmed that you need version 0.2.2.3? In other words, would 0.2.2 be sufficent?

Also, have you considered creating a conda package and uploading it to your personal channel on Anaconda Cloud? If this R package is not well-supported, it isn't a good fit for conda-forge (because it will be hard for us to maintain it).

Reply:
Yes, colearendt is the maintainer. I reached him via his email and requested a GitHub release. He said he will make the release but he did not, yet. I have to continue working on my PhD project, so I decided to use my own fork until his release. 0.2.2 is a very old version (from 2015), it would not be sufficient unfortunately. 

Would it be alright if we continue with my fork for now and I update the recipe once colearendt release it? Otherwise I can try Anaconda personal channel.

Reply:
@sbilge My preference would be for you to create a personal conda package until the maintainer releases a new version (preferably to CRAN, but at minimum a GitHub tag).

@conda-forge/r Any opinions on maintaining this recipe? I know I'm always the curmudgeon about maintaining non-CRAN packages...

Reply:
Well, the given URL is neither CRAN nor any upstream supported release URL (sometimes forks become the new upstream, but the commit history does not indicate that). So I don't think you are the curmudgeon here ;)

@sbilge you can upload the package to your own anaconda.org channel, the steps are something like:
* create an anaconda.org account (free)
* create an upload $token inside that account
* install `conda-build + anaconda-client`
* build+upload via `conda-build staged-recipes/recipes/r-tidyjson --token $token`
* Install/use the package via: `conda install -c $yourChannel r-tidyjson` or `conda install $yourChannel:r-tidyjson` or add it to your environment.yml.

That should not take that long... Would that be sufficient?

This PR can be updated and merged after upstream did tag a new release.

Reply:
@dbast @jdblischak Thank you. I will do so. Should I close the pull request till I update the PR with the original release?

Reply:
@sbilge Yes, please close it for now so that we know that it is not in need of our attention. You can reopen it once it's been updated for the new release. Thanks!